### PR TITLE
KOGITO-4343: Fix JSONIX model generation with gwt 2.9

### PR DESCRIPTION
**Pull request by treblereel**
JIRA: https://issues.redhat.com/browse/KOGITO-4343

Related PRs:
https://github.com/kiegroup/gwt-jsonix-schema-compiler/pull/11
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1606
https://github.com/kiegroup/kie-wb-common/pull/3570
https://github.com/kiegroup/kie-wb-common/pull/3562
https://github.com/kiegroup/kie-wb-common/pull/3582

to test localy set gwt-jsonix-schema-compiler version to 1.3.0-SNAPSHOT

@yesamer ^^

### CODE REVIEW

**Comment by yesamer** [comment]
@treblereel What is the difference with this PR? https://github.com/kiegroup/kie-wb-common/pull/3570

**Comment by tiagobento** [comment]
@yesamer The other one was not supposed to be there anymore. Can you please review this and the related PRs? cc @karreiro

**Review by yesamer — APPROVED** [review]
@treblereel LGTM, thank you!

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/widgets/grid/columns/factory/BaseSingletonDOMElementFactoryTest.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public void checkAttachDomElement() {
                      domElementOnDisplay);
     }
 
-    @Test
+    //@Test
```
Comment (yesamer) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/widgets/grid/columns/factory/BaseSingletonDOMElementFactoryTest.java:128]: @treblereel Why is this test ignored?

**Review by karreiro — COMMENTED** [review]
Thank you, @treblereel. The PR looks great! 🚀 Can we uncomment the test that @yesamer mentioned? :-)

**Comment by treblereel** [comment]
wrong branch has been pushed ... let me fix it

**Comment by treblereel** [comment]
java.io.IOException: No space left on device :(

**Review by karreiro — APPROVED** [review]
Thank you, @treblereel :-)

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/marshaller/marshall/DMNMarshallerTest.java [hunk 2]
```diff
@@ -278,81 +271,20 @@ public void testWithIncludedModelsWhenNodeParentIsDefinitions() {
...
-        dmnMarshaller.connect(diagram,
-                              dmnDiagramElementIds,
-                              definitionsStunnerPojo,
-                              dmnEdges,
-                              node,
-                              view);
-
-        verify(sourceConnection).isAuto();
-        verify(targetConnection).isAuto();
-    }
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/marshaller/marshall/DMNMarshallerTest.java:340]: Why we need to remove this test please?

**Review by jomarko — COMMENTED** [review]
What webapp should I use to verify this please?

I used **kie-wb-common/kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-testing**, however that has editor tab always empty. 

Could someone please confirm that is bug on master? not related tot his PR?

Screenshot from 2021-03-30 13-29-58

**Comment by jomarko** [comment]
No resources to review this anymore. Please proceed without my further review.

**Comment by yesamer** [comment]
@karreiro @treblereel I think this can close, considering Kogito asserts are no more present in this repo.

**Pull request closed without merging**